default_plugins/consoleme_default_plugins/plugins/metrics/metrics.py

```diff
@@ -1,5 +1,8 @@
 from typing import Dict, Optional, Union
 
+from consoleme.config import config
+from consoleme_default_plugins.plugins.metrics.cloudwatch import CloudWatchMetric
```

Comment:
Let's move this import down into the conditional below so we don't initialize the cloudwatch client unless we're going to use it

Reply:
Went with the importlib approach we discussed yesterday